.circleci/config.yml

```diff
@@ -1118,11 +1118,6 @@ jobs:
     resource_class: qkc/ax101
     steps:
       - checkout-with-mise
-      - run:
```

Comment:
Maybe just comment out, so it's easier when merging, I think it's how it was did previously.

Reply:
Agree. It would also be helpful to add comments explaining why this is commented out.